Source/Components/StackCluster.swift

```diff
@@ -35,11 +35,15 @@ extension ClusterLayout {
             let title = meta["title"] as? String
             let spacing = (meta["spacing"] as? NSNumber)?.floatValue
             let preserveParentWidth = meta["preserve_parent_width"] as? Bool
-            let axisRawValue = meta["axis"] as? Int
-            let axis = axisRawValue.flatMap { UILayoutConstraintAxis(rawValue: $0) }
+            let orientationRawValue = meta["orientation"] as? String
+            var axis = UILayoutConstraintAxis.vertical
             let hexString = meta["background_color"] as? String
             let backgroundColor = hexString.flatMap { UIColor(hexString: $0) }
-
+            
+            if "horizontal" == orientationRawValue {
```

Comment:
We should maybe introduce an enum with (`String`-valued) raw values for this and use the failable initializer to map strings onto enumeration cases.

Something like this:
```
var axis = UILayoutConstraintAxis.vertical
if let orientationRawValue = meta["orientation"] as? String,
   let orientation = Orientation(rawValue: orientationRawValue) {
     axis = orientation.layoutConstraintAxis
}
```
with
```
public enum ClusterLayout {
    
    public enum Orientation: String {
        case horizontal
        case vertical
    }
}

extension ClusterLayout.Orientation {
    
    var layoutConstraintAxis: UILayoutConstraintAxis {
        switch self {
        case .horizontal: return .horizontal
        case .vertical: return .vertical
        }
    }
}
```